resources/grafana/sources/rhacs-cluster-overview.json

```diff
@@ -1568,12 +1656,12 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sum(container_memory_working_set_bytes{namespace=~\"rhacs-$instance_id\", container=\"scanner\", job=~\"kubelet\"}) by (namespace) / sum(container_spec_memory_limit_bytes{namespace=~\"rhacs-$instance_id\", container=\"scanner\", job=~\"kubelet\"}) by (namespace)",
+          "expr": "sum(container_memory_working_set_bytes{namespace=~\"rhacs-$instance_id\", job=\"kubelet\", container=\"scanner\"}) by (namespace)",
```

Comment:
To be honest, it used to be percentage but I thought that maybe real values can be more useful because we might want to compare scanner, matcher and indexer in the transition period. I definitely want to return percentage once we switch to 4.5